etc/keepalived/start_slave_svcs.sh.j2

```diff
@@ -25,3 +25,12 @@ ssh -o StrictHostKeyChecking=no -i /etc/keepalived/openlab.pem "ubuntu@{{ zuul_s
         sudo systemctl start zuul-* --all
         sudo systemctl start apache2
     "
+
+# Get tools to change webhook or change the DNS record(Manually)
+if [ ! -f "/etc/keepalived/cli.py" ];then
+    wget -P /etc/keepalived/  https://raw.githubusercontent.com/wangxiyuan/Toy/master/github-app-webhook-url-update-tool/cli.py
```

Comment:
If possible we could move god-like-yuan's script into labkeeper, avoid to be removed by accidently.

Reply:
god-like-yuan ;-). Make sence to me.